### PR TITLE
ACMS-1705: regenerate site studio package with dependencies

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**

Fixes #[ACMS-1705](https://backlog.acquia.com/browse/ACMS-1705)

**Proposed changes**

This change help export drupal standard config and site studio configuration in separate directory and its provides drush post command that import/export site studio package on config import/export.

**Alternatives considered**
Manually do the configuration and run drush command separately.

**Testing steps**
export your site studio keys
`./vendor/bin/acms site:install --yes`
`./vendor/bin/drush sitestudio:package:export`
include `$settings['site_studio_sync'] = '../config/cohesion/sync';` in settings.php
verify all packages with dependencies are getting exported in site studio sync directory.

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by chandan-singh7929 — APPROVED** [review]
LGTM

**Review by apathak18 — APPROVED** [review]

File: modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_window.yml [hunk 1]
```diff
@@ -1554,7 +1559,7 @@ json_values: |
       }
   }
 json_mapper: '{}'
-last_entity_update: entityupdate_0036
+last_entity_update: entityupdate_0041
```
Comment (vishalkhode1) [inline modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_window.yml:1562]: @rajeshreeputra Please revert this back.

File: modules/acquia_cms_image/config/pack_acquia_cms_image/cohesion_templates.cohesion_content_templates.media_image_full.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ langcode: en
 status: true
 dependencies:
   config:
-    - cohesion_templates.cohesion_master_templates.master_template
+    - cohesion_templates.cohesion_master_templates.mstr_tpl_master_template_boxed
```
[the remark this answers is not in the document]
Comment (rajeshreeputra) [reply modules/acquia_cms_image/config/pack_acquia_cms_image/cohesion_templates.cohesion_content_templates.media_image_full.yml:6]: it is adding dependency to export cohesion_templates.cohesion_master_templates.mstr_tpl_master_template_boxed template and not using any of the template anywhere.

File: modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_button.yml [hunk 3]
```diff
@@ -481,15 +486,15 @@ json_values: |
                   "title": "Button ID",
                   "schema": {
                       "type": "string",
-                      "escape": true,
-                      "required": true
```
[the remark this answers is not in the document]
Comment (rajeshreeputra) [reply modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_button.yml:485]: This is now added OOTB hence not required to be added from config.
Screenshot 2023-03-29 at 9 34 02 AM

File: modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_window.yml (hunk 1 above)
Comment (rajeshreeputra) [reply modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_window.yml:1562]: updated

File: modules/acquia_cms_image/config/pack_acquia_cms_image/cohesion_templates.cohesion_content_templates.media_image_full.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vishalkhode1) [reply modules/acquia_cms_image/config/pack_acquia_cms_image/cohesion_templates.cohesion_content_templates.media_image_full.yml:6]: Looks good then.

File: modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_button.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (vishalkhode1) [reply modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_button.yml:485]: I assume the functionality is same as earlier. (Button text field is required like earlier)

**Review by vishalkhode1 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**